Ontology/v2/sbol-vo.rdf

```diff
@@ -47,25 +42,34 @@
   <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">notes</rdfs:label>
 </owl:AnnotationProperty>
 
+<owl:Class rdf:about="#Glyph">
+  <rdfs:subClassOf rdf:resource="http://www.w3.org/2002/07/owl#Thing"/>
+  <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Glyph</rdfs:label>
+</owl:Class>
+
+<owl:Class rdf:about="#InteractionGlyph">
+  <rdfs:subClassOf rdf:resource="#Glyph"/>
+  <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">InteractionGlyph</rdfs:label>
+</owl:Class>
+
+<owl:Class rdf:about="#InteractionNodeGlyph">
+  <rdfs:subClassOf rdf:resource="#Glyph"/>
+  <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">InteractionNodeGlyph</rdfs:label>
+</owl:Class>
+
+<owl:Class rdf:about="#MolecularSpeciesGlyph">
+  <rdfs:subClassOf rdf:resource="#Glyph"/>
+  <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">MolecularSpeciesGlyph</rdfs:label>
+</owl:Class>
+
+<owl:Class rdf:about="#SequenceFeatureGlyph">
+  <rdfs:subClassOf rdf:resource="#Glyph"/>
+  <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">SequenceFeatureGlyph</rdfs:label>
+</owl:Class>
+
 <owl:Class rdf:about="#InhibitionGlyph">
   <rdfs:subClassOf rdf:resource="#Glyph"/>
-  <rdfs:subClassOf>
-    <owl:Restriction>
-      <owl:onProperty rdf:resource="#isGlyphOf"/>
-      <owl:someValuesFrom rdf:resource="http://sbols.org/v2#Interaction"/>
-    </owl:Restriction>
-  </rdfs:subClassOf>
-  <rdfs:subClassOf>
-    <owl:Restriction>
-      <owl:onProperty rdf:resource="#isGlyphOf"/>
-      <owl:someValuesFrom>
-        <owl:Restriction>
-          <owl:onProperty rdf:resource="http://sbols.org/v2#type"/>
-          <owl:someValuesFrom rdf:resource="http://identifiers.org/sbo/SBO:0000169"/>
```

Comment:
This change seems to have removed all of the SBO restrictions for interactions; we don't want to lose that.

Reply:
The same is the case for all of the other interactions